recipes/hookee/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "hookee" %}
+  {% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6f7bac526a73f5295e584b4decef48e459cfeca4da4900a47a37eb96098dea6f
+
+build:
+  noarch: python
+  entry_points:
+    - hookee=hookee.cli:main
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - future
+    - pyngrok
+    - python-dotenv
+    - flask
+  run:
+    - python
+
+test:
+  imports:
+    - hookee
```

Comment:
Please test the cli:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

it would be something like:


```
test:
   commands:
      - hookee --help
```